src/models.py

```diff
@@ -748,6 +753,47 @@ def _single_sentence_forward(self, batch, task, predict):
                 _, out['preds'] = logits.max(dim=1)
         return out
 
+    def _pair_sentence_acceptability_forward(self, batch, task, predict):
+        out = {}
+
+        if task.name == 'cola-pair-frozen':
+            bs = get_batch_size(batch)
+            # embed the sentence
+            sent_embs, _ = self.sent_encoder(batch['input0'], task)
+            # pass to a task specific classifier
+            classifier = self._get_classifier(task)
+            index = batch['index'].squeeze(dim=1)
+            logits_full = classifier(sent_embs)[range(bs), index]
+            sent1_key = batch['input1']['bert_wpm_pretokenized'][range(bs), index] - 2
```

Comment:
this will break whenever you aren't using BERT tokenization?

Reply:
The program would not enter this part of the code, if it isn't using bert. (there won't be suitable task specific module for it, so it will get ValueError("Module not found for %s" % task.name)). This forward function is only for probing BERT with minimal pair, the name of this function did not reflect that, I should probably change it.

Reply:
Ah gotcha. Are you all planning on supporting non-BERT versions of this task? Otherwise renaming would be helpful.

Reply:
No. Minimal pairs is specially used as a tool to probe the properties of BERT model. I can't see a scenario where this task is necessary to understand something in other models.